Studio.kt

```diff
@@ -144,9 +144,9 @@ object Studio {
             onCloseRequest = { if (error != null) exitApplicationFn() else confirmClose() },
         ) {
             CompositionLocalProvider(LocalWindow provides window) {
-                val windowContext = WindowContext(window)
+                val windowContext = WindowContext.Compose(window)
+                MainWindowContent(windowContext)
```

Comment:
We've moved `MainWindowContent` out of the scope that provides `windowContext` as we already provide it in the body of `MainWindowContent`.